website/components/consul-on-kubernetes-hero/index.tsx

```diff
@@ -0,0 +1,68 @@
+import Button from '@hashicorp/react-button'
+import s from './style.module.css'
+
+interface Cta {
+  url: string
+  text: string
+}
+
+interface Media {
+  type: 'image' | 'video'
+  source: string
+  alt?: string
+}
+
+interface ConsulOnKubernetesHeroProps {
+  title: string
+  description: string
+  ctas: Cta[]
+  media: Media
+}
+
+export default function ConsulOnKubernetesHero({
+  title,
+  description,
+  ctas,
+  media,
+}: ConsulOnKubernetesHeroProps) {
+  return (
+    <div
+      className={s.ckHero}
+      style={{
+        backgroundImage: `url(${require('./images/background-design.svg')})`,
```

Comment:
This background image gets a little noisy on medium screens, but I don't have a recommendation on how to fix it; so I'll leave this for design review to snuff out.

Reply:
Agreed 👍